static/news/news.md

```diff
@@ -1,6 +1,18 @@
 ####Please read the [Wiki/FAQ](https://github.com/pymedusa/SickRage/wiki) before opening an issue####
 All issues and bug reports must be opened at [GitHub](https://github.com/pymedusa/SickRage/issues)
 
+####2017-02-19####
+
+## 0.1.11
+
+Little time has passed, but a lot of work has been done. With the most noticeable changes regarding user interface improvements. We'd like to thank wimpyrbx for his awesome work on the new logo, as well as the many designs he has provided which have resulted in a new look & feel. Helping shaping up Medusa to what it's now. You can expect to see even more UI improvements in the coming months.
+
+As for easier management of backlogged episodes we've added controls for starting a manual or forced search in the backlog overview page. Regarding managing backlogged episodes, you will now see a real time warning when changing the desired quality for a show. Meaning you will now get some help, trying to prevent a huge backlog, when you change the quality for a show.
```

Comment:
"Meaning you will now get some help, trying to prevent a huge backlog, when you change the quality for a show."

Replace that with this "You will now get a helpful message regarding episodes being added or removed to your backlog while changing the show quality settings."